src/client/app/containers/editor.ts

```diff
@@ -39,7 +39,11 @@ export class Editor implements AfterViewInit {
      * Initialize the component and subscribe to all the necessary actions.
      */
     async ngAfterViewInit() {
-        this._monacoEditor = await this._monaco.create(this._editor, { theme: 'vs' });
+        let _overrides = { theme: 'vs' };
+        if (/#\/view/.test(location.hash)) {
```

Comment:
This should be a parameter passed to Editor from "view" component and not looking at the location.hash.